.github/workflows/license-check.yml

```diff
@@ -0,0 +1,27 @@
+name: license-check
+
+on:
+  pull_request:
+    branches: [v2/*]
+  push:
+    branches: [v2/*]
+
+jobs:
+  license-check:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: v1.17.x
+    - name: Install go-licenses
+      run: |
+        go install github.com/google/go-licenses@latest
+    - uses: actions/checkout@v2
+    - name: Go mod vendor
+      run: |
+        go mod vendor
+    - name: Run go-licenses
+      run: |
+        go-licenses csv github.com/corazawaf/coraza/v2
```

Comment:
Is this checking `v2/master` only? What about PRs? Can we make the `vX` an argument?

Reply:
Could we enable Travis for checking PRs?

Reply:
Sorry, why do we want Travis if we have github actions?

Reply:
It will work, it is not like it's downloading coraza, it will use the local directory with the specified package.

@ShiMing-Q I wouldn't use Travis and GH actions, it would be duplicated work